src/boot/efi/util.c

```diff
@@ -390,10 +390,15 @@ void clear_screen(size_t attr) {
         ST->ConOut->ClearScreen(ST->ConOut);
 }
 
-void sort_pointer_array(
-                void **array,
+static inline void *array_element(const void *array, size_t stride, size_t i) {
```

Comment:
`static inline` only really makes sense in headers, since it gives the compiler the chance to decide whether to inline the code or not. But inside of .c files it's pointless, the compiler can optimize this on its own, hence please drop "inline" here.